Demo/Components/InlineFilterDemoViewController.swift

```diff
@@ -6,7 +6,7 @@
 
 final class InlineFilterDemoViewController: UIViewController {
     lazy var inlineFilterView: InlineFilterView = {
-        let view = InlineFilterView()
+        let view = InlineFilterView(frame: .zero)
```

Comment:
Can we use `InlineFilterView(withAutoLayout: true)`?

Reply:
Yep ;)